packages/vector_graphics_compiler/lib/src/geometry/path.dart

```diff
@@ -175,6 +192,79 @@ class CubicToCommand extends PathCommand {
   /// axis.
   final double y3;
 
+  /// A [Point] representation of [x3],[y3], the end point of the curve.
+  Point get endPoint => Point(x3, y3);
+
+  /// Subdivides the cubic curve described by [start], [control1], [control2],
+  /// [end].
+  ///
+  /// The returned list describes two cubics, where elements `0, 1, 2, 3` are
+  /// the start, cp1, cp2, and end points of the first cubic and `3, 4, 5, 6`
+  /// are the start, cp1, cp2, and end points of the second cubic.
+  static List<Point> subdivide(
+    Point start,
+    Point control1,
+    Point control2,
+    Point end,
+    double t,
+  ) {
+    final Point ab = Point.lerp(start, control1, t);
+    final Point bc = Point.lerp(control1, control2, t);
+    final Point cd = Point.lerp(control2, end, t);
+    final Point abc = Point.lerp(ab, bc, t);
+    final Point bcd = Point.lerp(bc, cd, t);
+    final Point abcd = Point.lerp(abc, bcd, t);
+    return <Point>[
+      start,
+      ab,
+      abc,
+      abcd,
+      bcd,
+      cd,
+      end,
+    ];
+  }
+
+  /// Precisions > 1 increase precision at cost of extra computation time.
+  double computeLength(Point start, [double precision = 2]) {
```

Comment:
The precision value is "good enough" to make one of customer: money's SVGs look equivalent to flutter_svg. A value of 1 is not good enough.

Reply:
Is it just computation time, or is it also binary size? We don't need to worry too much about computation time I'd hope

Reply:
Ahh I should remove this comment and fix the doc comment, I'll follow up.

It's more about computation time but there are probably cases where it increases binary size, I doubt by much.